src/components/MapComponent.vue

```diff
@@ -95,6 +138,35 @@ export default {
         }
       }
     },
+    myLocation() {
+      this.$store.commit("setMapCenter", this.currLocation);
+      this.myLocationAccessed = true;
+      this.$store.commit("setZoom", 18);
+      this.myLocationAccessed = false;
+    },
+    createModeOn() {
+      this.createPathMenuVisible = !this.createPathMenuVisible;
+      if (this.createPathMenuVisible) {
+        this.$refs.pathDiv.style.display = "grid";
```

Comment:
@JatinThakur14 Dont edit styles directly. Control this via a css class. And local variable.